cmd/kubetoken/kubetoken.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubetoken
+
+import (
+	"fmt"
+
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/spf13/cobra"
+)
+
+func KubeToken() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "kubetoken",
+		Short: "Gets a token to access the Kubernetes API with client authentication",
+	}
+
+	cmd.RunE = func(*cobra.Command, []string) error {
+		c, err := okteto.NewKubeTokenClient()
```

Comment:
I think we need to run the `okteto context` command first, so if variables like `OKTETO_USER`, `OKTETO_TOKEN,  or `OKTETO_URL` are defined, those are being used to initialize the context.

@pchico83 Do we want to support that as part of this command?

Reply:
yes, we do for most of the commands if not all. we should read from variables if they are available to initialize the okteto context

Reply:
I've added the call to okteto context but I'm not sure if the parameters are ok. Could you check? 

cc @AdrianPedriza @jLopezbarb

Reply:
Yes, you're doing it right. The thing is if the context exists in the kubeconfig file but not in okteto context it will fail, but I think we are showing an error for that

Reply:
> if the context exists in the kubeconfig file but not in okteto context it will fail

No problem, that should never happen since this command will be ran after `okteto kubeconfig`

Reply:
That makes me think why is a command itself then. If it's a command it will appear in the help information, and any user could use it before running `okteto kubeconfig`. Should we return a proper error with a CTA to run `okteto kubeconfig` first?

Reply:
We are hiding the command in order to not promote its usage. Do you think that's enough? In general, the user should never type it into their terminal, it should all be transparent with the [new kubeconfig usage](https://github.com/okteto/okteto/pull/3409)

Reply:
If it's part of `kubeconfig` command, maybe we could add a hidden flag to the `kubeconfig` command. That way if the user types the command will always work, no matter if they have it configured or not

Reply:
@jLopezbarb it's not under `kubeconfig`but could be. The naming could be changed if you find something more appropriate.

As it is now, the command is ran with `okteto kubetoken` and it's hidden so that `okteto --help` won't show it